README.adoc

```diff
@@ -113,23 +111,14 @@ cd keep-core
 KEEP_ETHEREUM_PASSWORD="password" ./keep-core --config configs/config.local.1.toml relay request
 ```
 
-== Relay maintainer
-
-To interact with the system properly, you should also run a relay-maintainer
-instance by doing:
-```
-./run-testnet-relay.sh
-```
-
```

Comment:
Sorry for asking, but I missed why are we removing the relay maintainer part?

Reply:
Because we don't use testnet relay with private (regtest) BTC network.

Reply:
Oh, because of the constant difficulty on the private btc network, right?

Reply:
Yeah exactly.